### ISSUE TITLE
Disable Facial Tracking while user is muted

### ISSUE THREAD

**Issue by dfgHiatus** [Neos-Metaverse/NeosPublic#2274]
## Is your feature request related to a problem? Please describe.
Currently, the Vive Facial Tracker sends information irrespective of voice mode. When I mute myself, my avatar continues to "mouth out" words and expressions, which can be frustrating.

While I understand this may be a desirable behavior for select circumstances (such as mouthing out words for sign language), a user still could accomplish this in any voice mode.

## Relevant issues
Not that I could find on the Github or Discord.

## Describe the solution you'd like
A toggle for the Vive Facial Tracker that disables lip tracking when a user is muted. This could be a component, or perhaps something internal.

## Describe alternatives you've considered
A LogiX script that gets my current voice mode, and nulls the MouthTrackingStreamManager when I mute myself. While simple enough to create, it would be nice to have this natively implemented.

**Comment by dfgHiatus** [comment]
Thinking more, it would be preferable for users to choose whether they would want to use this sort of system. I emphasize the importance of an opt-in component, or toggle.

**Comment by shiftyscales** [comment]
Mm- I'm unsure about this one- namely considering that the mouth tracker does a lot more than just mouthing words in terms of facial expression/controls.

What is it specifically you find frustrating about your avatar mouthing what you're saying, @dfgHiatus? It would help a lot if we could identify a potential lower-level concern/issue here if you can think of one.

**Comment by Frooxius** [comment]
If you want it as a setting, it'll have to wait until the Settings UI is reworked. We could change the default for now if it's warranted, but it would be one or the other.

**Comment by H3BO3** [comment]
I don't think changing the default would be a good idea. I'm sure there are mute users who rely on things as they are right now. A settings toggle that had to be switched on would be a good idea when the settings UI is reworked imo.

**Comment by dfgHiatus** [comment]
I can't say there's a significant lower-level issue here @shiftyscales. I agree in the sense that a mouth tracker enables much more than speech - and for mute users this is a must have.

To give an example of an issue I've had, I'll need to take a phone call or have to speak with someone in the same room. Knowing my mouth moves when I don't want it to isn't more than a minor inconvenience, but I imagine I am not the only user whose run into this situation. 

We should see if enough other users have the same problem before committing ourselves.

@Frooxius would it be feasible to instead implement this as a component? I imagine it could be put on the same slot as the AvatarExpressionDriver/VisimeDriver, and that way users could opt-in to the system if they'd want to.

**Comment by Frooxius** [comment]
@dfgHiatus I can add LogiX node to determine if the user is recording voice message or not, which is going to be general enough and let you do that solution itself.

However doing it this way won't prevent the face data from actually being sent over network if that's what you're concerned with though the way it does with voice data, that will still require proper native solution.

**Comment by dfgHiatus** [comment]
So I understand you, does this only apply to in-game voice messages? This solves part of the problem, I can work with this

**Comment by Frooxius** [comment]
Yes it'll output true anytime you're recording a voice message - when the voice message visual is shown around your head.
